src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -51,31 +72,42 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton safeTxHash={txDetails.detailedExecutionInfo.safeTxHash} />
         </div>
       )}
+      {txData?.operation === Operation.DELEGATE && (
+        <div className="tx-operation">
+          <DelegateCallWarning isKnown={isCustomTxInfo(txInfo) && !!txInfo?.to?.name} />
+        </div>
+      )}
+      {isMultiSendTxInfo(txInfo) && (
+        <>
+          <TxInfoMultiSend txInfo={txInfo} />
+          <br />
+        </>
+      )}
+
       {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
       {safeTxHash && <TxDataRow title="SafeTxHash:" value={safeTxHash} inlineType="hash" hasExplorer={false} />}
       {created && <TxDataRow title="Created:" value={formatDateTime(created)} />}
       <TxDataRow title="Executed:" value={executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE} />
-      <br></br>
+      <br />
 
       {/* Advanced TxData */}
-      {txData?.operation !== undefined && (
-        <TxDataRow title="Operation:" value={`${txData.operation} (${Operation[txData.operation].toLowerCase()})`} />
-      )}
-      {safeTxGas && <TxDataRow title="safeTxGas:" value={safeTxGas} />}
-      {baseGas && <TxDataRow title="baseGas:" value={baseGas} />}
-      {gasPrice && <TxDataRow title="gasPrice:" value={gasPrice} />}
-      {gasToken && <TxDataRow title="gasToken:" value={gasToken} inlineType="hash" />}
-      {refundReceiver && <TxDataRow title="refundReceiver:" value={refundReceiver} inlineType="hash" />}
-      {confirmations?.length > 0 && (
-        <TxDataRow title="Signatures:" value={signaturesFromConfirmations} inlineType="rawData" />
-      )}
-      {txData?.hexData && <TxDataRow title="Raw data:" value={txData.hexData} inlineType="rawData" />}
-      {txData?.operation === Operation.DELEGATE && (
-        <div className="tx-operation">
-          <DelegateCallWarning isKnown={isCustomTxInfo(txInfo) && !!txInfo?.to?.name} />
-        </div>
-      )}
-      {isMultiSendTxInfo(txInfo) && <TxInfoMultiSend txInfo={txInfo} />}
+      <StyledButtonLink onClick={toggleExpanded} color="primary" iconSize="sm" textSize="xl">
+        Advanced Details
+      </StyledButtonLink>
+      <CollapsibleSection show={expanded}>
+        {txData?.operation !== undefined && (
+          <TxDataRow title="Operation:" value={`${txData.operation} (${Operation[txData.operation].toLowerCase()})`} />
+        )}
+        {safeTxGas && <TxDataRow title="safeTxGas:" value={safeTxGas} />}
+        {baseGas && <TxDataRow title="baseGas:" value={baseGas} />}
+        {gasPrice && <TxDataRow title="gasPrice:" value={gasPrice} />}
```

Comment:
Is this the same for EIP-1559 txns?

Reply:
Yes. I guess it should be refactored. 👍

Reply:
This are the fields that we receive from the backend. For `EIP-1559` compliant chains we are not receiving different fields

Reply:
Gotcha, thanks.